theme_buzzy/views/snippets/s_references.xml

```diff
@@ -4,7 +4,7 @@
 <template id="s_references" inherit_id="website.s_references">
```

Comment:
Commit message:
The list of modules (`*: theme_buzzy,...`) should be the first thing to appear in the commit message :wink: So invert the two paragraphs :+1: